glossary.rst

```diff
@@ -121,7 +121,7 @@ Glossary
 
    YAML
         *YAML* is a recursive acronym for "YAML Ain't a Markup Language". It's a
-        lightweight, humane data serialization language used extensively in
+        lightweight, human data serialization language used extensively in
```

Comment:
Leaving it alone does seem reasonable; however, I am wondering if it would be easier to just say "human friendly" vs "humane". Also after looking at [yaml.org](http://www.yaml.org/), the definition you provided for "humane" seems to fit.